src/sentry/conf/api_pagination_allowlist_do_not_modify.py

```diff
@@ -108,4 +108,5 @@
     "UserSubscriptionsEndpoint",
     "UserUserRolesEndpoint",
     "VstsSearchEndpoint",
+    "OrganizationWorkflowStatsEndpoint",
```

Comment:
i don't think our chart components can handle paginated APIs? cc @ameliahsu

Reply:
I double checked on this and that seems correct, we don't need pagination.